From b2d71fec19c5613e322a166c335a846add25a32b Mon Sep 17 00:00:00 2001
From: Duanqiang Wen <duanqiangwen@net-swift.com>
Date: Mon, 14 Nov 2022 14:31:52 +0800
Subject: [PATCH] openeuler: net: ngbe add build support
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: b2d71fec19c5613e322a166c335a846add25a32b
Modified-by-SEL: No


driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I61PSD
CVE: NA

add build support for ngbe module

Signed-off-by: Duanqiang Wen <duanqiangwen@net-swift.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/net/ethernet/netswift/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/netswift/Makefile b/drivers/net/ethernet/netswift/Makefile
index 0845d08600be..5690b6392ce2 100644
--- a/drivers/net/ethernet/netswift/Makefile
+++ b/drivers/net/ethernet/netswift/Makefile
@@ -4,3 +4,4 @@
 #
 
 obj-$(CONFIG_TXGBE) += txgbe/
+obj-$(CONFIG_NGBE) += ngbe/
-- 
2.33.0

